relay/nakama/signer/signer.go

```diff
@@ -0,0 +1,20 @@
+package signer
+
+// signer.go manages the creation and loading of the Nakama private key used to sign
+// all transactions.
+
+import (
+	"context"
+
+	"pkg.world.dev/world-engine/sign"
+)
+
+type Signer interface {
+	SignTx(ctx context.Context, personaTag string, namespace string, data any) (*sign.Transaction, error)
+	SignSystemTx(ctx context.Context, namespace string, data any) (*sign.Transaction, error)
+	SignerAddress() string
```

Comment:
Is it necessary to provide SignerAddress via a getter? Looks like we could do this by making it a public var in NakamaSigner and TxSigner if we wanted to, but I'm guessing this means that the interface will not require a public var named `SignerAddress`, but maybe that's fine? 

No big deal since it's only one time and cuz there is a relevant interface though.

Reply:
There's no way to include a public var in an interface; interfaces can only contain methods. 

There's also no straightforward way to reach through an interface to the underlying implementation to get a specific field. You would have to do a type assertion.